### PR TITLE
Add 'omega' field to documents

### PR DESCRIPTION
**Pull request by mrakitin**
Work on 2021-08-31 with @tacaswell, @RobertSchaffer1, @JunAishima, @mskinner5278.

### CODE REVIEW

File: mxtools/flyer.py [hunk 1]
```diff
@@ -56,15 +56,21 @@ def describe_collect(self):
...
         }
         return return_dict
 
     def collect(self):
         self.unstage()
 
         now = ttime.time()
-        data = {f"{self.detector.name}_image": self._datum_ids[0]}
+        self._master_metadata = self._extract_metadata()
+        data = {f"{self.detector.name}_image": self._datum_ids[0], "omega": self._master_metadata}
```
Comment (RobertSchaffer1) [inline mxtools/flyer.py:73]: Is it possible for this to get called before self._datum_ids gets populated? If so, should we catch this exception?

**Comment by RobertSchaffer1** [comment]
All told, I think it's fine (do we LGTM here?).

File: mxtools/flyer.py (hunk 1 above)
Comment (mrakitin) [reply mxtools/flyer.py:73]: The `_asset_docs_cache` is populated with the `resource` dictionary once the detector is staged, which happens on the `kickoff` step, so it happens much earlier and in the predefined order (`kickoff` -> `complete` -> `collect`. The `self._datum_ids` is populated in the `collect_asset_docs` method, which is called before `collect`, so we should be good here. Also, it's not a new code, it's just the reformatting that caused it to look like a new code after we added the `omega` key.

**Comment by mrakitin** [comment]
> All told, I think it's fine (do we LGTM here?).

@RobertSchaffer1, thanks for the review. Just approve it if you are happy with the changes. Thanks!

**Comment by RobertSchaffer1** [comment]
I didn't mean to re-request a review, sorry for the confusion. When you say approve it, do you mean merge, or is there some other mechanism I can't find?

**Comment by mrakitin** [comment]
I meant to go to the "Files changed" tab, and use the <kbd>Review changes</kbd> button to leave your review and approval:
image

**Review by RobertSchaffer1 — APPROVED** [review]

**Comment by RobertSchaffer1** [comment]
Ah thanks, done. I'm not used to github (used gitlab before, but it has a slightly different flow).

**Review by mskinner5278 — APPROVED** [review]

**Review by JunAishima — APPROVED** [review]
looks good!

**Pull request merged**